crates/re_types/definitions/rerun/blueprint/components/visual_bounds2d.fbs

```diff
@@ -13,7 +13,7 @@ namespace rerun.blueprint.components;
 struct VisualBounds2D (
   "attr.docs.unreleased",
   "attr.rerun.scope": "blueprint",
-  "attr.rust.derive": "Copy, PartialEq, bytemuck::Pod, bytemuck::Zeroable",
+  "attr.rust.derive": "Default, Copy, PartialEq, bytemuck::Pod, bytemuck::Zeroable",
```

Comment:
I think this is a case where we want an explicit `Default`. I don't know what the default is of `Range2D` right now (and it could change in the future), but I can imagine it being one of several things (0-0, -∞—+∞, +∞—-∞, NaN-NaN), neither of which seem like a great default for `VisualBounds2D`

Reply:
Sound reasonable 👍. Hard to pick a good value though 🤔 
Btw. this is one of the many ones where the `Default` is kinda always wrong. We practically always want the (upcoming) `CompondentFallbackProvider` to do something clever - in this case it will be providing the current bounding box.